### ISSUE TITLE
The try statement inside the Text is also Formatted.

### ISSUE THREAD

**Issue by shira-shun** [nicklockwood/SwiftFormat#1395]
```swift
let input = #"let foo = "\(try bar())""#
// let foo = "try \(bar()) <-Build Error
testFormatting(for: input, rule: FormatRules.hoistTry)
```

When hoistTry is enabled, try statements inside the text are formatted and cannot be built.

**Comment by nicklockwood** [comment]
@shira-shun fixed in 0.51.3

**Comment by shira-shun** [comment]
@nicklockwood 

```swift
let text = """
abc
\(try bar())
xyz
"""
```

Sorry, it seems to be a false positive in cases like this...

**Comment by nicklockwood** [comment]
@shira-shun fixed in 0.51.4
